tests/regression/gdoc/test_chrome_gdoc_create_isolate_image_1.py

```diff
@@ -5,7 +5,7 @@ class TestSikuli(PerfBaseTest):
 
     def setUp(self):
         self.pre_run_script = "test_chrome_gdoc_read_blank_page"
-        self.pre_run_script_test_url_id = "TEST_TARGET_ID_BLANK_PAGE"
+        self.pre_run_script_test_url_id = self.global_config['gsuite']['gdoc-blank-page']
```

Comment:
According to @ShakoHo, it was used for debugging by thinker. It is for achieving something like the input latency measurement we have now. I suggest that you can remove these 6 files.